content/django-vs-flask.md

```diff
@@ -0,0 +1,100 @@
+# Django vs Flask: Which should You Choose For Your Project?
+[Django](https://www.djangoproject.com/) and [Flask](https://flask.palletsprojects.com/en/1.1.x/) are among the most popular Python web frameworks.
+In this article, we're going to review their main strengths and weaknesses to help you decide which one is a better fit for your project.
+
+
+# Django
+[![django-logo](images/flask-vs-django-django-logo.png)](https://www.djangoproject.com/)
+## Approach
+Django opts for an all batteries-included approach.
+From a powerful ORM to an authentication system or a templating engine, Django takes care of it all without the need to learn any extra libraries.
+
+It also has the concept of pluggable apps, which means you can drop an app folder inside your project and get its functionality right away with very little configuration. 
+
+## Main Features
+- Django has a very powerful ORM that exposes a lot of SQL to the python programming language.
+- It has Authentication and Authorization built-in, without the need for any extra configuration.
+- It also boasts a built-in admin site, and
+ a built-in mailing system.
+
+## Pitfalls
+- Being tightly coupled with its toolset, Django can be difficult to use in combination with other tools, like replacing the form library, for example.
+- While Django's **default scaffolding function** is not bad for small projects, it does not scale without modification when trying to manage things such as multiple environments.
+- This functionality implements the Model View Template pattern but requires some effort to implement other architecture types with it.
+
+## Developer Experience
+One of the main perks of using an all batteries included approach, is that the developer experience will be more or less the same across the framework.
+This proves very beneficial to people new to the framework, as they can follow a pattern and a guide to follow good practices by looking at existing code.
```

Comment:
```suggestion
This can be very beneficial for users new to the framework as they can follow a pattern, and guide, for best practices by looking at existing code.
```